scene/gui/tab_bar.cpp

```diff
@@ -881,6 +881,8 @@ void TabBar::_update_hover() {
 		if (hover != -1) {
 			emit_signal(SNAME("tab_hovered"), hover);
 		}
+
+		_update_cache();
```

Comment:
`_update_cache()` seems to call `_update_hover()`. Isn't there a risk of infinite recursion?

Reply:
This `_update_cache()` is only called when `hover != hover_now` and that is taken care of right in the next line, so I can't see a recursion happening.